packages/runtime-dom/src/jsx.ts

```diff
@@ -406,7 +406,7 @@ export interface DataHTMLAttributes extends HTMLAttributes {
 
 export interface DetailsHTMLAttributes extends HTMLAttributes {
   open?: Booleanish
-  onToggle?: Event
+  onToggle?: (payload: ToggleEvent) => void
```

Comment:
I used the tooling in place to fix this locally on my project. Dont know if it is good practice or not.
```ts
interface DetailsHTMLEvents {
    onToggle?: ToggleEvent;
}
export interface DetailsHTMLAttributes extends HTMLAttributes, EventHandlers<DetailsHTMLEvents> {
    open?: Booleanish;
}
```